core/templates/domain/utilities/url-interpolation.service.ts

```diff
@@ -23,7 +23,6 @@ import { Injectable } from '@angular/core';
 import { AlertsService } from 'services/alerts.service';
 import { UrlService } from 'services/contextual/url.service';
 import { UtilsService } from 'services/utils.service';
-import { LoggerService } from 'services/contextual/logger.service';
```

Comment:
I noticed that this PR removes a lot of imports, which is great. 

One question: is this due to the changes you've made, or were these unused imports which were not previously detected? If the latter, maybe worth adding this to #8423 as a lint check?

Reply:
Nope it wasn't, I had just so happen to notice it.

Reply:
OK, thanks! Added to #8423.